stable/hlf-peer/README.md

```diff
@@ -107,7 +107,7 @@ The following table lists the configurable parameters of the Hyperledger Fabric
 | `secrets.peer.tlsRootCert`         | TLS root CA certificate: as 'cert.pem'               | ``                                                         |
 | `secrets.peer.tlsClient`           | TLS client secret: as 'tls.crt' and 'tls.key'        | ``                                                         |
 | `secrets.peer.tlsClientRootCerts`  | TLS Client root CA certificate files (any name)      | ``                                                         |
-| `secrets.channel`                  | Secret containing Channel tx for peer to create/join | ``                                                         |
+| `secrets.channel`                  | Array of secrets containing channel creation file    | ``                                                         |
```

Comment:
Should be `channels`

Reply:
i.e. `secrets.channels`

`Array of secrets containing channel creation file` was already correct.